tests/chains/tendermint/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG_VER=1.20.5
+ARG GOLANG_VER=1.21
```

Comment:
Please specify a patch version for making tests reproducible.
`1.21.8` looks good at the moment.

Reply:
Use 1.21.8
https://github.com/hyperledger-labs/yui-relayer/pull/134/commits/afb578e9cde1aed70e69e63c9d2ef24a13bc27aa